ods_ci/tests/Tests/1000__model_serving/1009_model_serving_triton_on_keserve/1009_model_serving_triton_kserve.robot

```diff
@@ -0,0 +1,89 @@
+*** Settings ***
+Documentation     Suite of test cases for OVMS in Kserve
+Library           OperatingSystem
+Library           ../../../libs/Helpers.py
+Resource          ../../../Resources/Page/ODH/JupyterHub/HighAvailability.robot
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHModelServing.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataConnections.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/ModelServer.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDashboardSettingsRuntimes.resource
+Resource          ../../../Resources/Page/ODH/Monitoring/Monitoring.resource
+Resource          ../../../Resources/OCP.resource
+Resource          ../../../Resources/CLI/ModelServing/modelmesh.resource
+Suite Setup       Triton On Kserve Suite Setup
+Suite Teardown    Triton On Kserve Suite Teardown
+Test Tags         Kserve
+
+*** Variables ***
+${INFERENCE_REST_INPUT_PYTORCH}=    @tests/Resources/Files/triton/rest_resnet50_input.json
+${PRJ_TITLE}=    model-serving-triton-project-resnet50
```

Comment:
iirc, this is too long and would make the route creation failing because of k8s limits. Similar tracked issue: https://issues.redhat.com/browse/RHODS-8921 
Pls double check if this is the case